automation/src/main/java/org/wso2/testgrid/automation/Test.java

```diff
@@ -90,17 +95,67 @@ public TestScenario getTestScenario() {
         return testScenario;
     }
 
+    /**
+     * Returns the Pre scenario script.
+     *
+     * @return location of the script as a String.
```

Comment:
We do not need to specify `as a String` here since it is already shown in the JavaDoc

Reply:
Done